app/src/androidTest/java/ch/sdp/vibester/fragment/WelcomeScreenFragmentTest.kt

```diff
@@ -57,83 +58,77 @@ class WelcomeActivityTest {
         return mockUser
     }
 
-    @Before
-    fun setUp() {
-        hiltRule.inject()
-        Intents.init()
-    }
-
     @After
     fun clean() {
         Intents.release()
     }
 
-    @Test
-    fun checkIntentOnPlay() {
-        val intent = Intent(ApplicationProvider.getApplicationContext(), WelcomeActivity::class.java)
-        createMockAuthenticatorLoggedOut()
-        val scn: ActivityScenario<WelcomeActivity> = ActivityScenario.launch(intent)
-
-        onView(withId(R.id.welcome_play)).perform(click())
-        intended(hasComponent(GameSetupActivity::class.java.name))
+    @Before
+    fun setup() {
```

Comment:
is there a reason for changing the name from setUp to setup?

Reply:
fixed